__tests__/fetcher/api_mockdata.ts

```diff
@@ -0,0 +1,2851 @@
+            ],
+          },
+        },
+      },
+      {
+        navigation: {
+          path: {
+            nodes: [
+              {
+                label: 'Mathematik',
+                url: '/mathe',
+              },
+              {
+                label: 'Mittelschule (Hauptschule)',
+                url: '/mathe/deutschland/bayern/mittelschule',
+              },
+              {
+                label: 'Klasse 7',
+                url: '/mathe/deutschland/bayern/mittelschule/klasse-7',
+              },
+              {
+                label: 'Rationale Zahlen ',
+                url:
+                  '/mathe/deutschland/bayern/mittelschule/klasse-7/proportionalität/rationale-zahlen',
+              },
+            ],
+          },
+        },
+      },
+      {
+        navigation: {
+          path: {
+            nodes: [
+              {
+                label: 'Mathematik',
+                url: '/mathe',
+              },
+              {
+                label: 'Alle Themen',
+                url: '/mathe/5',
+              },
+              {
+                label: 'Deutschland',
+                url: '/mathe/deutschland',
+              },
+              {
+                label: 'Bayern',
+                url: '/mathe/deutschland/bayern',
+              },
+              {
+                label: 'Mittelschule M-Zug',
+                url: '/mathe/deutschland/bayern/mittelschule-m-zug',
+              },
+              {
+                label: 'Klasse 7',
+                url: '/mathe/deutschland/bayern/mittelschule-m-zug/klasse-7',
+              },
+              {
+                label: 'Dezimalbrüche',
+                url:
+                  '/mathe/deutschland/bayern/mittelschule-m-zug/klasse-7/dezimalbrüche',
+              },
+            ],
+          },
+        },
+      },
+      {
+        navigation: {
+          path: {
+            nodes: [
+              {
+                label: 'Mathematik',
+                url: '/mathe',
+              },
+              {
+                label: 'Alle Themen',
+                url: '/mathe/5',
+              },
+              {
+                label: 'Deutschland',
+                url: '/mathe/deutschland',
+              },
+              {
+                label: 'Schleswig-Holstein',
+                url: '/mathe/deutschland/schleswig-holstein',
+              },
+              {
+                label: 'Gymnasium G9',
+                url: '/mathe/deutschland/schleswig-holstein/gymnasium-g9',
+              },
+              {
+                label: '6. Klasse',
+                url:
+                  '/mathe/deutschland/schleswig-holstein/gymnasium-g9/6.-klasse',
+              },
+              {
+                label: 'Dezimalzahlen',
+                url:
+                  '/mathe/deutschland/schleswig-holstein/gymnasium-g9/6.-klasse/dezimalzahlen',
+              },
+            ],
+          },
+        },
+      },
+    ],
+  },
```

Comment:
```suggestion
  taxonomyTerms: {
    nodes: [
      {
        navigation: {
          path: {
            nodes: [
              { label: 'Mathematics', url: '/math' },
              { label: 'All topics', url: '/math/1' },
            ],
          },
        },
      },
      {
        navigation: {
          path: {
            nodes: [
              { label: 'Mathematics', url: '/math' },
              { label: 'Class 6' url: '/math/class-6' },
            ],
          },
        },
      },
    ],
  },
```

A self made example is better than a real one. When it is shorter it is much easier for me to understand the logic behind the structure. Most often you can say in tests, that N=2 is a list :smile: N=1 is also often enough :smile: